stumpy/stamp.py

```diff
@@ -12,11 +12,12 @@ def _mass_PI(
     T,
     M_T,
     Σ_T,
-    T_subseq_isconstant,
     trivial_idx=None,
     excl_zone=0,
     left=False,
     right=False,
+    T_subseq_isconstant=None,
+    Q_subseq_isconstant=None,
```

Comment:
`Q_subseq_isconstant` was missing here. So, I added it. In addition, I put the isconstant params at the end so that the order of arguments becomes more similar to what we have in `core.mass`.